modules/network-observability-loki-install.adoc

```diff
@@ -5,7 +5,7 @@
 :_content-type: PROCEDURE
 [id="network-observability-loki-installation_{context}"]
 = Installing the Loki Operator
-It is recommended to install link:https://catalog.redhat.com/software/containers/openshift-logging/loki-rhel8-operator/622b46bcae289285d6fcda39[Loki Operator version 5.7], This version provides the ability to create a LokiStack instance using the `openshift-network` tenant configuration mode. It also provides fully automatic, in-cluster authentication and authorization support for Network Observability.
+It is recommended to install link:https://catalog.redhat.com/software/containers/openshift-logging/loki-rhel8-operator/622b46bcae289285d6fcda39[Loki Operator version 5.7]. This version provides the ability to create a LokiStack instance using the `openshift-network` tenant configuration mode. It also provides fully automatic, in-cluster authentication and authorization support for Network Observability.
```

Comment:
Do we want to call out 5.7 specifically or specify something like `5.7+`?

Reply:
5.7+ is probably what we want.